manifests/service.pp

```diff
@@ -11,4 +11,11 @@
     hasstatus => true,
     restart   => $::postfix::params::restart_cmd,
   }
+  if $::osfamily == 'RedHat' {
+    exec { 'alternatives --set mta /usr/sbin/sendmail.postfix':
```

Comment:
Is there maybe a resource type that does that instead of an exec?

Reply:
There is, but I didn't want to introduce an unnecessary module dependency. Do you want me to rebuild this patch with a requirement that the [alternatives](https://github.com/voxpupuli/puppet-alternatives) module be installed?

Reply:
I'll submit that approach in a separate PR.

Reply:
Submitted #195